modules/facts/lib/facter/busid.rb

```diff
@@ -0,0 +1,5 @@
+Facter.add(:busid) do
```

Comment:
This fact name should probably be something more descriptive like `gpu_device_bus_id` or at least `gpu_busid` since it lives in a global namespace.

Reply:
3f32d61aef1436912f127e974f9dec054b4eaa14